scripts/load_data.sh

```diff
@@ -0,0 +1,2 @@
+curl -L -o public/data/v2/getTermDump/allTerms.json searchneu.com/data/v2/getTermDump/allTerms.json
```

Comment:
Adding a script for just grabbing the prod data we need for indexing.

Reply:
There was some code that did a similar thing before we moved to elasticsearch. This code ran every time before the files were needed, and only downloaded them if the local files didn't exist before re-downloading them. Perhaps you restore some of this code and run it in a [jest `beforeAll`](https://jestjs.io/docs/en/setup-teardown)? With this, we wouldn't need the new `fetch` command too.

https://github.com/ryanhugh/searchneu/blob/5c790dd74fc2edf7cedf635fe0b43efb86c99a68/backend/server.js#L177-L215

But this code looks good too! Feel free to merge :)